README.md

```diff
@@ -75,13 +75,18 @@ e.g.
 There are two additional tasks to enable development within an electron app natively:
 
 1. `npm run start:architect:electron`: to start the webpack dev server
-  - Note: must be running on port 3003.
+
+- Note: must be running on port 3003.
+
 1. `npm run start:network-canvas:electron`: to start the webpack dev server
-  - Note: must be running on port 3000.
-2. `npm run electron:dev` (in another terminal session)
-  1. Copies the electron source to `./electron-dev`
-  2. Runs the electron app from there
 
+- Note: must be running on port 3000.
+
+2. `npm run preelectron:dev` (in another terminal session)
```

Comment:
I think we should make it clear that this step only needs to happen when you first set up the repo, or when you bump the version number.